### PR TITLE
feature/spec: Added test cases for EC2 plugins

### PR DESCRIPTION
**Pull request by AkhtarAmir**

### CODE REVIEW

File: plugins/aws/ec2/openDocker.spec.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                "FromPort": 0,
+                "IpProtocol": "tcp",
+                "IpRanges": [
+                    {
+                        "CidrIp": "0.0.0.0/0"
+                    }
+                ],
+                "Ipv6Ranges": [],
+                "PrefixListIds": [],
+                "ToPort": 65535,
```
Comment (dipsubha06) [inline plugins/aws/ec2/openDocker.spec.js:28]: Docker port is 'tcp': [2375, 2376]. But here in the test cases those are not open.

File: plugins/aws/ec2/openFTP.spec.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                "FromPort": 0,
+                "IpProtocol": "tcp",
+                "IpRanges": [
+                    {
+                        "CidrIp": "0.0.0.0/0"
+                    }
+                ],
+                "Ipv6Ranges": [],
+                "PrefixListIds": [],
+                "ToPort": 65535,
```
Comment (dipsubha06) [inline plugins/aws/ec2/openFTP.spec.js:28]: same here FTP port is 20 and 21. But in failing case we dont have test cases with these two open

**Review by dipsubha06 — CHANGES_REQUESTED** [review]
In open port test cases please check if the failing test case is having the concerned ports open.

File: plugins/aws/ec2/openDocker.spec.js (hunk 1 above)
Comment (AkhtarAmir) [reply plugins/aws/ec2/openDocker.spec.js:28]: I am sending all ports open which are 0-65535 so technically that covers 'tcp': [2375, 2376] as well.

File: plugins/aws/ec2/openFTP.spec.js (hunk 2 above)
Comment (AkhtarAmir) [reply plugins/aws/ec2/openFTP.spec.js:28]: I am sending all ports open which are 0-65535 so technically that covers 'tcp': [20, 21] as well.

**Review by giorod3 — APPROVED** [review]

**Pull request merged**